linux_os/guide/services/obsolete/r_services/package_rsh-server_removed/rule.yml

```diff
@@ -20,6 +20,7 @@ severity: high
 identifiers:
     cce@rhel6: 27062-9
     cce@rhel7: 27342-5
+    cce@rhel8: 82184-3
```

Comment:
@matejak I am not seeing this package for RHEL8. Should we remove it for RHEL8? Or should we have it enabled still "just in case"? Probably should check the other packages in the list as well.

Reply:
Good catch. The rule was in a RHEL8 profile, so I have subsequently removed it from the product and profile in a separate commit.

Reply:
@matejak do you mind checking to see if the other package_*_removed in this PR exist in RHEL8 or EPEL8?

Reply:
This package_removed cleanup started to interfere with the OSPP profile, so I propose not to solve those in the PR.